datastew/repository/base.py

```diff
@@ -31,6 +31,11 @@ def get_all_mappings(self, limit=1000) -> List[Mapping]:
         """Get all embeddings up to a limit"""
         pass
 
+    @abstractmethod
+    def get_terminology_specific_mappings(self, terminology_name, limit=1000) -> List[Mapping]:
+        """Get all embeddings from a specified terminology up to a limit"""
```

Comment:
I think it makes sense to already refactor this at this point to:

`get_all_mappings` + `get_terminology_specific_mappings` to:

```python
def get_mappings(self, terminology_name = None, limit=1000):
    if terminology_name is None:
        # get_allmappings
    else:
        get_terminology_specific_mappings
```

Reply:
I implemented this but the if clause you have is different, I can change the code if you'd like to.

Reply:
Also fine!